README.md

```diff
@@ -64,7 +64,35 @@ Then you will see something like this:
 To get ruff automatically running install the following Plugin:
 - [VSCode - Ruff Plugin](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff)
 
-### Continuous Delivery
+
+## Testing
+
+A test suite of a typical Python package consists of unit, integration, and documentation tests
+(i.e. the code snippets embedded in the documentation),
+and it is truly important to write the tests to ensure that the code works as intended.
+
+The package template provides a minimal setup and the tests can be executed by invoking the Pytest runner:
+
+```shell
+pytest
+```
+
+The runner looks for tests in `src` and `tests` folders. On top of that, the Python code
+in the top-level `README.md` file is also executed, so we can demonstrate
+the most imporant functionality:
+
+```python
+>>> from project_name.foo import foo
+>>> foo()
+True
+
+```
```

Comment:
The way you phrased it, it sounds like the code snipped is imporant. However, I think what you actually wanted to make is an example on how to write a doctest, right?

So, maybe:

```suggestion
the most imporant functionality. For example snippets like these will be executed by pytest:

```python
>>> from project_name.foo import foo
>>> foo()
True

```
```